### DISCUSSION TITLE
2023-08-01

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#222] [daily_meeting]
### 개인 일정
노아 - 3~4시 근로

#### 다같이
~~- API 변경 포인트 논의~~
~~- 쿠키 관리 정책 재논의~~
~~- 백엔드에서 response에서 set cookie를 통해 memberId 설정해준다.~~
~~- API 변경 포인트 확정~~
~~-API 변경사항 반영~~

### 내일의 할 일

- API 변경 사항 개발

### 언젠가 할 일
- 소켓, 폴링 양방향 통신 어떻게 할지
- 기획 변경된 이유 정리
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)
- A라는 스터디를 참여후 B스터디 참여한 뒤 다시 A스터디에 참여할 시 닉네임을 찾지 못한 이슈(로그인이 된다면 해결될 문제이긴 함)

**Comment by MoonJeWoong** [comment]
### 체크인
9점 어제 논의했던 사항 중 모호한게 해결된 것 같아서 기분이 좋습니다! 오늘 하루도 잘 마무리 해봐요~

### 체크아웃
10점 길고 길었던 API와의 전쟁을 1차적으로 잘 마무리한 것 같아서 기분 좋습니다!

**Reply by aak2075** [reply]
형식 맞춰주세요

**Comment by nlom0218** [comment]
### 체크인

10점 - 웹팩 설정 글 3개 마무리 😁

### 체크아웃

9점 - 시간이 느므 빨리 가유 😭

**Comment by aak2075** [comment]
### 체크인

10점 - 벌써 8월이네요 이번 달도 화이팅합시다~

### 체크아웃

10점 - Api깔끔하게 수정돼서 만족합니다

**Comment by jaehee329** [comment]
### 체크인
8점 - 무난무난혀요 잘 해보죠~

### 체크아웃
7점 - 회의가 안 무난무난하네요.. ㅎㅎ

**Comment by woosung1223** [comment]
### 체크인
8점 - 오늘따라 유난히 일어나기 힘들었어서 -2점

### 체크아웃
8점 - API 설계가 그래도 만족스럽게 된 것 같아서 다행입니다

**Comment by yeopto** [comment]
### 체크인

7점 - SO SO 합니다 오늘

### 체크아웃

7점 - 아무런 생각이 없습니다..

**Comment by woo-jk** [comment]
### 체크인
3점 - 생략

### 체크아웃
7점 - 컨디션이 좀 별루다
